src/Components/RingdownCard.js

```diff
@@ -8,37 +8,102 @@ import Drawer from './Drawer';
 import Ringdown from '../Models/Ringdown';
 import RingdownDetails from './RingdownDetails';
 import RingdownEta from './RingdownEta';
+import Alert from './Alert';
 import './RingdownCard.scss';
 
-function RingdownCard({ className, ringdown }) {
+function RingdownCard({ className, ringdown, onStatusChange }) {
   const [isExpanded, setExpanded] = useState(false);
+  const [showCancel, setShowCancel] = useState(false);
+  const [showRedirect, setShowRedirect] = useState(false);
+  const [currentHospital] = useState(ringdown.hospital.id);
+
+  function handleCancel() {
+    setShowCancel(false);
+    onStatusChange(ringdown, Ringdown.Status.CANCEL_ACKNOWLEDGED);
+  }
+
+  function handleRedirect() {
+    setShowRedirect(false);
+    onStatusChange(ringdown, Ringdown.Status.REDIRECT_ACKNOWLEDGED);
+  }
 
   return (
     <Card
       className={classNames(
         'ringdown-card',
         { 'ringdown-card--offloaded': ringdown.currentDeliveryStatus === Ringdown.Status.OFFLOADED },
+        { 'ringdown-card--cancelled': ringdown.currentDeliveryStatus === Ringdown.Status.CANCELLED },
+        {
+          'ringdown-card--cancelled':
+            ringdown.currentDeliveryStatus === Ringdown.Status.REDIRECTED &&
+            ringdown.hospital.id &&
+            ringdown.hospital.id === currentHospital,
+        },
         className
       )}
       header={
         isExpanded && ringdown.currentDeliveryStatus !== Ringdown.Status.OFFLOADED ? null : `Incident #${ringdown.dispatchCallNumber}`
       }
       body={isExpanded && ringdown.currentDeliveryStatus !== Ringdown.Status.OFFLOADED ? null : ringdown.chiefComplaintDescription}
     >
+      {ringdown.currentDeliveryStatus === Ringdown.Status.CANCELLED && (
+        <span className="ringdown-eta ringdown-card__eta">
+          <span className="ringdown-eta__prefix">Cancelled: </span>
+          {DateTime.fromISO(ringdown.timestamps[Ringdown.Status.CANCELLED]).toLocaleString(DateTime.TIME_24_WITH_SECONDS)}
```

Comment:
I don't see the timestamp included as part of the design. The styling is also not matching what I see in Figma:

<img width="846" alt="Screen Shot 2021-08-17 at 11 16 16 AM" src="https://user-images.githubusercontent.com/140766/129780102-7c30a48b-b9b1-4630-b55d-fea39b607eb7.png">